src/ResourceManager/LogicApp/Commands.LogicApp.Test/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,50 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System.Reflection;
+using System.Runtime.InteropServices;
+using Xunit;
+
+// General Information about an assembly is controlled through the following 
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("Microsoft.Azure.Commands.LogicApp.Test")]
+[assembly: AssemblyDescription("Azure LogicApp PowerShell cmdlet test assembly")]
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("Microsoft")]
+[assembly: AssemblyProduct("Microsoft.Azure.Commands.LogicApp.Test")]
+[assembly: AssemblyCopyright(Microsoft.WindowsAzure.Commands.Common.AzurePowerShell.AssemblyCopyright)]
+[assembly: AssemblyTrademark("")]
+[assembly: AssemblyCulture("")]
+
+// Setting ComVisible to false makes the types in this assembly not visible 
+// to COM components.  If you need to access a type in this assembly from 
+// COM, set the ComVisible attribute to true on that type.
+[assembly: ComVisible(false)]
+
+// The following GUID is for the ID of the typelib if this project is exposed to COM
+[assembly: Guid("de7887f1-5984-42eb-96a0-8730673c04a8")]
+
+// Version information for an assembly consists of the following four values:
+//
+//      Major Version
+//      Minor Version 
+//      Build Number
+//      Revision
+//
+// You can specify all the values or you can default the Build and Revision Numbers 
+// by using the '*' as shown below:
+// [assembly: AssemblyVersion("1.0.*")]
+[assembly: AssemblyVersion(Microsoft.WindowsAzure.Commands.Common.AzurePowerShell.AssemblyVersion)]
+[assembly: AssemblyFileVersion(Microsoft.WindowsAzure.Commands.Common.AzurePowerShell.AssemblyFileVersion)]
```

Comment:
[Fixed]

From: Hovsep [mailto:notifications@github.com]
Sent: Monday, February 1, 2016 3:25 PM
To: Azure/azure-powershell <azure-powershell@noreply.github.com>
Cc: Pankaj Singh Negi <Pankaj.Negi@microsoft.com>
Subject: Re: [azure-powershell] Azure Logic App PowerShell Cmdlets (#1753)


In src/ResourceManager/LogicApp/Commands.LogicApp.Test/Properties/AssemblyInfo.cs<https://github.com/Azure/azure-powershell/pull/1753#discussion_r51498602>:

> +

> +// The following GUID is for the ID of the typelib if this project is exposed to COM

> +[assembly: Guid("de7887f1-5984-42eb-96a0-8730673c04a8")]

> +

> +// Version information for an assembly consists of the following four values:

> +//

> +//      Major Version

> +//      Minor Version

> +//      Build Number

> +//      Revision

> +//

> +// You can specify all the values or you can default the Build and Revision Numbers

> +// by using the '*' as shown below:

> +// [assembly: AssemblyVersion("1.0.*")]

> +[assembly: AssemblyVersion(Microsoft.WindowsAzure.Commands.Common.AzurePowerShell.AssemblyVersion)]

> +[assembly: AssemblyFileVersion(Microsoft.WindowsAzure.Commands.Common.AzurePowerShell.AssemblyFileVersion)]

your module will not be a part of RDFE package. Please have the same version string here as you have in the src/ResourceManager/LogicApp/AzureRM.LogicApp.psd1 file.

—
Reply to this email directly or view it on GitHub<https://github.com/Azure/azure-powershell/pull/1753/files#r51498602>.